course/assignments/R-intro-lectures-assignments.md

```diff
@@ -0,0 +1,60 @@
+# Week 1 (4 marks) Intro to programming
+1. Why is it beneficial to use a programming language rather than a spreadsheet software for data analyses?
+2. Get setup with Rstudio at home (or a lab computer after hours). Install R, Rstudio, and the necessary packages (tidyverse, rmarkdown). Use `install.packages(<package_name>)`. Load all the libraries you installed into your environment with `library(<package_name>)` and send us the output of running `sessionInfo()`.
```

Comment:
I also think one of the fun things with markdown is to generate professional looking pdfs easily, so it might be an encouraging task to include =)